.github/workflows/deploy_main_image.yaml

```diff
@@ -0,0 +1,44 @@
+# This workflow creates docker image based on main branch and publishes it to project's container registry.
+# It doesn't use any cache and performs clean build, so it may take longer than other jobs.
+
+on: 
+  push:
+    branches:
+      - 'main'
+jobs:
+  deploy-main-image:
+    name: Build and deploy main docker image
+    runs-on: ubuntu-latest
+    env:
+      PLATFORM: linux/amd64,linux/arm64
+      MAIN_IMAGE: ghcr.io/${{ github.repository_owner }}/rover-soft:stable
+
+    steps:
+      - name: lowercase image name
+        run: |
+          echo "MAIN_IMAGE=${MAIN_IMAGE,,}" >>${GITHUB_ENV}
```

Comment:
Is this necessary and is this the best way to do it?

Reply:
I am afraid so. Docker requires image repository to be in lowercase. There is no way to transform the string in a workflow expression, so I have to do it in separate step.

Reply:
Alright, it can stay then. Maybe it would be good to add an explanation that our username has uppercase letters and ghcr isn't happy about it, so noone removes this in the future?